DataFormats/simulation/include/SimulationDataFormat/GeneratorHeader.h

```diff
@@ -0,0 +1,89 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \author R+Preghenella - August 2017
+
+#ifndef ALICEO2_DATAFORMATS_GENERATORHEADER_H_
+#define ALICEO2_DATAFORMATS_GENERATORHEADER_H_
+
+#include <string>
+#include <map>
+
+namespace o2
+{
+namespace dataformats
+{
+
+class GeneratorInfo;
+class CrossSectionInfo;
+class HeavyIonInfo;
+
+/*****************************************************************/
+/*****************************************************************/
+
+class GeneratorHeader
```

Comment:
could you add short class descriptions as documentation? (Adding info such as if the user is expected to extend the class ... which would justify use of pointers)